app/workers/users/record_field_test_event_worker.rb

```diff
@@ -8,6 +8,7 @@ class RecordFieldTestEventWorker
     def perform(user_id, goal)
       @user = User.find_by(id: user_id)
       return unless @user
+      return if FieldTest.config["experiments"].nil?
 
       FieldTest.config["experiments"].each_key do |key|
```

Comment:
@jeremyf What are your thoughts on this approach? I've usually used it because `nil.to_h` returns `{}` and calling `each` on an empty `Enumerable` is a no-op.

```suggestion
      FieldTest.config["experiments"].to_h.each_key do |key|
```